schema_salad/makedoc.py

```diff
@@ -593,6 +593,9 @@ def main():  # type: () -> None
 
     args = parser.parse_args()
 
+    makedoc(args)
+
+def makedoc(args):
```

Comment:
schema_salad/makedoc.py:596: error: Call to untyped function "makedoc" in typed context

Reply:
Ohh that's going to be fun, I need to annotate that whole set of code.